core/src/Revolution/modX.php

```diff
@@ -2981,4 +2981,89 @@ public function _postProcess() {
         }
         $this->invokeEvent('OnWebPageComplete');
     }
+
+    /**
+     * Removes unwanted tags and/or tag attributes from an HTML string
+     *
+     * @param string $htmlSource The html string to clean
+     * @param string|array $allowedTags An array or comma-separated list of tag names to allow
+     * @param string|array $allowedAttr An array or comma-separated list of tag attribute names to allow
+     */
+    public function stripHTML(string $htmlSource, $allowedTags = '', $allowedAttr = ''): string
```

Comment:
Would it be possible to add a Unit Test for this with various test cases?

Reply:
Yes, I will likely need a little guidance creating the tests (at least initially) as it'll be my first time doing so. I've been wanting to get some experience with UTs so I definitely don't want to pass it off ;-) I'll throw some questions to you on Slack via DM...

Reply:
https://docs.modx.com/3.x/en/contribute/code/running-tests